tasks/src/text-generation/data.ts

```diff
@@ -6,11 +6,11 @@ import { TASKS_MODEL_LIBRARIES } from "../const";
 const taskData: TaskData = {
 	datasets: [
 		{
-			description: "A large multilingual dataset of web text. Used to pretrain generative models in various languages.",
+			description: "A large multilingual dataset of crawled text from the web. It is used to train generative models in various languages.",
```

Comment:
I'd say 
A large multilingual dataset of text crawled from the web. Used to train generative models in various languages.

We need to keep them short :')

Reply:
Yes, also now I realize the second sentence might be redundant given this is the generation page

Reply:
Perfect! Will apply both comments